bitnami/minio/README.md

```diff
@@ -0,0 +1,179 @@
+| `image.tag`                          | MinIO image tag                                                                              | `{VERSION}`                                             |
+| `image.pullPolicy`                   | Image pull policy                                                                            | `Always`                                                |
+| `image.pullSecrets`                  | Specify docker-registry secret names as an array                                             | `[]` (does not add image pull secrets to deployed pods) |
+| `image.debug`                        | Specify if debug logs should be enabled                                                      | `false`                                                 |
+| `clientImage.registry`               | MinIO Client image registry                                                                  | `docker.io`                                             |
+| `clientImage.repository`             | MinIO Client image name                                                                      | `bitnami/minio-client`                                  |
+| `clientImage.tag`                    | MinIO Client image tag                                                                       | `{VERSION}`                                             |
+| `mode`                               | MinIO server mode (`standalone` or `distributed`)                                            | `standalone`                                            |
+| `statefulset.replicaCount`           | Number of pods (only for Minio distributed mode). Should be 4 <= x <= 32                     | `4`                                                     |
+| `statefulset.updateStrategy`         | Statefulset update strategy policy                                                           | `RollingUpdate`                                         |
+| `statefulset.podManagementpolicy`    | Statefulset pods management policy                                                           | `Parallel`                                              |
+| `existingSecret`                     | Existing secret with MinIO credentials                                                       | `nil`                                                   |
+| `accessKey.password`                 | MinIO Access Key. Ignored if existing secret is provided.                                    | _random 10 character alphanumeric string_               |
+| `accessKey.forcePassword`            | Force users to specify an Access Key                                                         | `false`                                                 |
+| `secretKey.password`                 | MinIO Secret Key. Ignored if existing secret is provided.                                    | _random 40 character alphanumeric string_               |
```

Comment:
It would be nice if we can have `usePasswordFile` just like we have with PostgreSQL and Redis https://github.com/helm/charts/tree/master/stable/postgresql